tools/tests/linter/fixtures/folder_validators/namespaces/invalid_folder/dup_path_d.yaml

```diff
@@ -0,0 +1,3 @@
+paths:
+  '/nodes/{metric}': {}
+  '/nodes/{node_id}': {}
```

Comment:
We should also account for when the paths appear in different namespace files.

Reply:
https://github.com/opensearch-project/opensearch-api-specification/pull/416/commits/b3da76c27c337fce57973102e77e65eb8ed75635